src/css/json-editor.css

```diff
@@ -0,0 +1,7 @@
+.grafana-json-datasource-editor {
+    width: 100%;
+}
+
+.grafana-json-datasource-editor div {
```

Comment:
I want to test whether this rule is required. If not we can also remove above rule and use something like `w100` class instead.